simpl/server.py

```diff
@@ -0,0 +1,158 @@
+# Copyright 2013-2015 Rackspace US, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Bottle Server Module."""
+
+import logging
+import os
+
+import bottle
+
+LOG = logging.getLogger(__name__)
+
+
+class EventletLogFilter(object):  # pylint: disable=R0903
+
+    """Receives eventlet log.write() calls and routes them.
+
+    Thread and WSGI messages only get written in debug mode.
+    API calls get written as INFO and to an access_log if provided.
+
+    An instance of this class can be passed in to a bottle.run command using
+    the `log` keyword.
+    """
+
+    def __init__(self, conf, log, access_log=None):
```

Comment:
Please add `param/keyword` docs for these parameters. I could make a guess at what `log` is, but it's not clear what the others are.

Also, `conf/self.config` doesn't appear to be used anywhere.